src/components/Editor/SearchBar.js

```diff
@@ -304,6 +305,7 @@ class SearchBar extends Component<Props, State> {
           handleNext={e => this.traverseResults(e, false)}
           handlePrev={e => this.traverseResults(e, true)}
           handleClose={this.closeSearch}
+          ref={c => (this.$input = c)}
```

Comment:
i agree that this feels odd

Reply:
@jasonLaster Instead of using a `ref` to directly call a method on a child component, would using redux here be a better option?  If so, can we very briefly pair on it?

Reply:
Refs aren't that weird anymore they're a pretty common pattern. But I would suggest adding a `isFocused` prop to `SearchInput` and handling the toggling of that value in the state here.